examples/tutorials/07_vpp-connection/main.go

```diff
@@ -0,0 +1,169 @@
+//  Copyright (c) 2019 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package main
+
+import (
+	"github.com/ligato/vpp-agent/plugins/vpp/binapi/interfaces"
+	"log"
+	"net"
+	"time"
+
+	"git.fd.io/govpp.git/api"
+	"github.com/ligato/vpp-agent/plugins/govppmux"
+
+	"github.com/ligato/cn-infra/agent"
+)
+
+func main() {
+	// Create an instance of our plugin.
+	p := new(HelloWorld)
+	p.GoVPPMux = &govppmux.DefaultPlugin
+
+	// Create new agent with our plugin instance.
+	a := agent.NewAgent(agent.AllPlugins(p))
+
+	// Run starts the agent with plugins, wait until shutdown
+	// and then stops the agent and its plugins.
+	if err := a.Run(); err != nil {
+		log.Fatalln(err)
+	}
+}
+
+// HelloWorld represents our plugin.
+type HelloWorld struct {
+	vppChan api.Channel
+
+	GoVPPMux govppmux.API
+}
+
+// String is used to identify the plugin by giving it name.
+func (p *HelloWorld) String() string {
+	return "HelloWorld"
+}
+
+// Init is executed on agent initialization.
+func (p *HelloWorld) Init() (err error) {
+	log.Println("Hello World!")
+
+	if p.vppChan, err = p.GoVPPMux.NewAPIChannel(); err != nil {
+		panic(err)
+	}
+
+	p.syncVppCall()
```

Comment:
These should be in `AfterInit`, because that is the last thing to execute, after all other plugins have `Init` and `AfterInit` completed. 

Actually even there it's questionable, because the `AfterInit` should return to let the cn-infra agent know that it completed okay.